BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -143,19 +143,12 @@ extension ScrollView {
 
 fileprivate final class ScrollerWrapperView: UIView {
     
-    let scrollView = UIScrollView()
+    let scrollView = CustomScrollView()
 
     private var refreshControl: UIRefreshControl? = nil {
 
         didSet {
```

Comment:
Rather than duplicating storage, I'd just forward this property to the scroll view:

```
var refreshControl: UIRefreshControl? {
    get { scrollView.refreshControl }
    set { scrollView.refreshControl = newValue }
}
```

Reply:
Or, actually – this is a private `var`. Should it be removed, and callers can just use `scrollView.refreshControl`?

Reply:
Most just saves on typing `scrollView.refreshControl` inside the `apply`? 

Since it's not doing any additional logic anymore, probably more explicit to use `scrollView.refreshControl` though.